create-projects/Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+
+
+
+
+def PROJECT_ID = env.project_id;
+def NEXUS_HOST = "https://nexus-cd.192.168.56.101.nip.io/";
+def PROJECT_ADMINS = env.project_admin;
+def PROJECT_GROUPS = "";
+def PIPELINE_TRIGGER_SECRET = env.PIPELINE_TRIGGER_SECRET;
+def sharedLibraryRepository
+def dockerRegistry
+
+
+node {
+  sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+  dockerRegistry = env.DOCKER_REGISTRY
+}
+def conts = containerTemplate( 
+            name: 'jnlp',
+            image: "${dockerRegistry}/cd/jenkins-slave-base",

```

Comment:
Consider pointing to the next version: `"${dockerRegistry}/cd/jenkins-slave-base:2"`